Utils/Services/ImagePickerService.swift

```diff
@@ -96,6 +111,37 @@ public final class MediaPickerService: NSObject, MediaPickerServiceType {
         }
     }
     
+    public func presentImagePickerController(from source: [UIImagePickerControllerSourceType],
+                                             for media: [MediaPickerMediaType],
+                                             _ onPermissionNotGranted: @escaping () -> Void) {
+        guard source.count > 1 else {
+            presentImagePickerController(from: source[0], for: media, onPermissionNotGranted)
+            return
+        }
+        
+        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+        
+        for option in source {
+            let button = UIAlertAction(title: option.menuTitle, style: .default, handler: { [unowned self] (action) -> Void in
+                option.isPermitted().startWithResult { [unowned self] in
```

Comment:
this block inside of the closure is exactly the same as the other method, make a function so you dont repeat code